webfront/serializers/taxonomy.py

```diff
@@ -204,33 +211,70 @@ def to_headers_representation(instance):
                 "accession": instance.accession,
                 "name": instance.full_name,
                 "children": instance.children,
-                "parent": instance.parent.accession
-                if instance.parent is not None
-                else None,
+                "parent": (
+                    instance.parent.accession if instance.parent is not None else None
+                ),
                 "source_database": "uniprot",
             }
         }
         return obj
 
     @staticmethod
-    def get_counters(instance, searcher, sq):
-        return {
-            "entries": searcher.get_number_of_field_by_endpoint(
-                "taxonomy", "entry_acc", instance.accession, sq
-            ),
-            "structures": searcher.get_number_of_field_by_endpoint(
-                "taxonomy", "structure_acc", instance.accession, sq
-            ),
-            "proteins": searcher.get_number_of_field_by_endpoint(
-                "taxonomy", "protein_acc", instance.accession, sq
-            ),
-            "sets": searcher.get_number_of_field_by_endpoint(
-                "taxonomy", "set_acc", instance.accession, sq
-            ),
-            "proteomes": searcher.get_number_of_field_by_endpoint(
-                "taxonomy", "proteome_acc", instance.accession, sq
-            ),
+    def can_use_taxonomy_per_entry(filters):
+        for key in filters:
+            if filters[key] != {} and key not in ["entry", "taxonomy"]:
+                return False
+        if "accession" in filters["entry"]:
+            return True
+        return False
```

Comment:
```suggestion
        for key, value in filters.items():
            if key not in ["entry", "taxonomy"] and value:
                return False

        return "accession" in filters["entry"]
```

An empty dictionary (`{}`) evaluates to `False`.